src/ctap/apdu.rs

```diff
@@ -177,10 +178,7 @@ impl TryFrom<&[u8]> for APDU {
             let extended_apdu_le_len: usize = payload
                 .len()
                 .checked_sub(extended_apdu_lc + 3)
-                .unwrap_or(0xff);
-            if extended_apdu_le_len > 3 {
-                return Err(ApduStatusCode::SW_WRONG_LENGTH);
-            }
```

Comment:
Don't you still want the length check? (And if yes, and this goes undetected, maybe add a unit test?)

Reply:
fixed with 35bdfe90ed52a5fac37d84b1ea6702b7d8c78001

Reply:
Ah, this was handled in the case below: `extended_apdu_le_len <= 3`
My oversight!